packages/frontend-v2/pages/testuseplan.tsx

```diff
@@ -0,0 +1,50 @@
+import { NextPage } from "next";
+import { LocalStorageKey, useLocalStorage } from "../hooks/useLocalStorage";
+import { usePlan } from "../hooks/usePlan";
+import { useStudentWithPlans } from "../hooks/useStudentWithPlans";
+
+const TestUsePlan: NextPage = () => {
+
+  // Perform localStorage action
+  const [tokenInStorage,] = useLocalStorage(LocalStorageKey.token, "");
+
+  const {student, error: studentError, isLoading: studentIsLoading} = useStudentWithPlans(tokenInStorage);
+  if (studentError) {
+    return (
+      <h1>student could not be fetched</h1>
+    )
+    // handle error
+  } else if (studentIsLoading) {
+    // handle loading
+  }
+  return (
+    <>
+      { student?.plans.map(plan => (
```

Comment:
Just a suggestion: I would even add a case to check if `student == null`, then can display something like "No student data". But up to you if you want to change it.